### PR TITLE
pyworkflow tests moved to root and has a plugin structure --> pyworkf…

### PR DESCRIPTION
**Pull request by pconesa**
…lowtests

scipion.py removed
Core version increased
Consider None values at Scalar comparison

### CODE REVIEW

File: pyworkflow/__init__.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 # if plugins are still compatible, so future hot fixes releases or even micros
 # or minor release should not change this CORE_VERSION. Only, when a new release
 # will break existing plugins, this number needs to be incremented.
-CORE_VERSION = '2.0'
+CORE_VERSION = '3.0'
```
Comment (delarosatrevin) [inline pyworkflow/__init__.py]: Can we use '3.0.0' to keep the same numbering format? Although I know '3.0' is equivalent to '3.0.0' in semantic versioning

File: pyworkflowtests/tests/test_object.py [hunk 2]
```diff
@@ -30,11 +30,10 @@
 
 import pyworkflow.object as pwobj
 import pyworkflow.tests as pwtests
-import pyworkflow.utils as pwutils
+from objects import Complex, MockSetOfImages, MockImage, MockObject, \
```
Comment (delarosatrevin) [inline pyworkflowtests/tests/test_object.py]: Please, follow PEP8, don't use \ to break lines

File: pyworkflow/__init__.py (hunk 1 above)
Comment (pconesa) [reply pyworkflow/__init__.py]: Ok

File: pyworkflowtests/tests/test_object.py (hunk 2 above)
Comment (pconesa) [reply pyworkflowtests/tests/test_object.py]: Thanks @delarosatrevin , I never tried the "python -m unittest discover" command.

Now it works too. I've addressed the other issues.

**Comment by pconesa** [comment]
I've accidentally pushed the changes to devel.... @delarosatrevin is it ok to leave it.

**Pull request merged**